### PR TITLE
Issue 56

### PR DESCRIPTION
**Pull request by kaeldai**
The posterior (pp_dnm) I originally found with their sample was 0.127147, not 0.170505 like they did. Also dnm was assuming the null hypothesis phenotypes were AT/AA/AG. I think this small difference is due to missing some of the I16 values.

With the fix the new posterior probability of mutation is 0.990036. It's not 1.0 exactly due to the fact we can't have a 0 phred-score and a bunch of really tiny calculations are adding up (whereas in the R script they just skipped those calculations).

dnm should no be more compatible within v4.1 and greater of the VCF standard. There is a new test case in testdata/trio/

### CODE REVIEW

**Comment by kaeldai** [comment]
Actually the new posterior is 1.0. The test data was slightly off.

**Comment by reedacartwright** [comment]
Jenkins, ok to test

File: src/parser.h [hunk 1]
```diff
@@ -35,6 +35,9 @@
 #define MIN_MAPQ 40
 #define MAX_QCALL_LINE 2048
 
+// maximum phred-scaled likelihood, as close to zero as possible (8 bit)
+#define MAX_PL 2255
```
Comment (reedacartwright) [inline src/parser.h]: Typo?

Comment (kaeldai) [inline src/parser.h]: Yes, but one that doesn't effect the code.

**Comment by kaeldai** [comment]
Yes, but one that doesn't effect the code

**Pull request closed without merging**